lib/views/home_page.dart

```diff
@@ -17,58 +17,97 @@ class _HomePageState extends State<HomePage> {
-              textAlign: TextAlign.center,
-              style: textStyleHookSubtitle(),
-            ),
-            const SizedBox(
-              height: 40,
-            ),
-            const Placeholder(
-              fallbackHeight: 180,
-            ),
-            const SizedBox(
-              height: 40,
-            ),
-            RichText(
-              text: TextSpan(
-                text: tr("homeWaiting"),
-                style: textStylePageSubtitle(),
-                children: [
-                  TextSpan(
-                    text: " ${tr("homeWaitingAct")}",
-                    style: textStylePageSubtitleImportant(),
-                  ),
-                  const TextSpan(
-                    text: " !",
-                  ),
-                ],
+        child: SingleChildScrollView(
+          child: Column(
+            children: [
+              const TitleHeader(),
+              const SizedBox(
+                height: 40,
               ),
-              textAlign: TextAlign.center,
-            ),
-            const SizedBox(
-              height: 30,
-            ),
-            ModulesCarousel(modules: modules),
-          ],
+              Text(
+                tr("homeEmergency"),
+                style: textStyleHookTitle(context),
+              ),
+              /* const SizedBox(
+                height: 5,
+              ),
+              Text(
+                tr("homeClickEmergency"),
+                textAlign: TextAlign.center,
+                style: textStyleHookSubtitle(context),
+              ), */
+              const SizedBox(
+                height: 40,
+              ),
+              Container(
+                width: MediaQuery.of(context).size.width * 0.8,
+                constraints: const BoxConstraints(
+                  minHeight: 180,
+                ),
+                padding: const EdgeInsets.all(15),
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                  borderRadius: BorderRadius.circular(10),
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.black.withOpacity(0.25),
+                      spreadRadius: 1,
+                      blurRadius: 7,
+                      offset: const Offset(2, 2),
+                    ),
+                  ],
+                ),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceAround,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    Text(
+                      'Le saviez-vous ?',
+                      textAlign: TextAlign.center,
+                      style: textStyleLearnCardTitle(context),
+                    ),
+                    const SizedBox(
+                      height: 10,
+                    ),
+                    Text(
+                      'Pour les malentendants le numéro d\'urgence est le 114.',
```

Comment:
hard coded text, please translate

Reply:
This will be dynamic in a near future by creating a card of "Did you know ?" elements which will be in a JSON for instance